src/BPEssentials/Commands/Kit.cs

```diff
@@ -0,0 +1,59 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BPEssentials.ExtensionMethods.Cooldowns;
+using BrokeProtocol.Entities;
+using BrokeProtocol.Utility;
+using System.Linq;
+
+namespace BPEssentials.Commands
+{
+    public class Kit : Command
+    {
+        public void Invoke(ShPlayer player, string kit)
+        {
+            if (!Core.Instance.KitHandler.List.Any(x => x.Name == kit))
+            {
+                player.TS("expFileHandler_error_notFound", player.T(Core.Instance.KitHandler.Name), kit);
+                return;
+            }
+            var obj = Core.Instance.KitHandler.List.FirstOrDefault(x => x.Name == kit);
+            if (!player.svPlayer.HasPermission($"{Core.Instance.Info.GroupNamespace}.{Core.Instance.KitHandler.Name}.{kit}"))
+            {
+                player.TS("expFileHandler_error_noPermission", player.T(Core.Instance.KitHandler.Name), kit);
+                return;
+            }
+            if (obj.Disabled)
+            {
+                player.TS("expFileHandler_error_disabled", player.T(Core.Instance.KitHandler.Name), kit);
+                return;
+            }
+            if (player.IsCooldown(Core.Instance.KitHandler.Name, kit))
+            {
+                player.TS("expFileHandler_error_cooldown", player.T(Core.Instance.KitHandler.Name), player.GetCooldown(Core.Instance.KitHandler.Name, kit).ToString());
+                return;
+            }
+            if (obj.Price > 0)
+            {
+                if (player.MyMoneyCount() < obj.Price)
+                {
+                    player.TS("expFileHandler_error_price", player.T(Core.Instance.KitHandler.Name), obj.Price.ToString(), player.MyMoneyCount().ToString());
+                    return;
+                }
+                player.TransferMoney(DeltaInv.RemoveFromMe, obj.Price, true);
+            }
+            foreach (var item in obj.Items)
+            {
+                player.TransferItem(DeltaInv.AddToMe, item.Id, item.Amount, true);
+            }
```

Comment:
I suggest changing this to something like `obj.GiveItems(player);`. public method inside the kit itself.